src/api/query.rs

```diff
@@ -109,12 +133,18 @@ pub fn block_hash_from_string(hash: &str) -> ApiResult<BlockHash> {
     })
 }
 
-/// Helper type for providing a way to represent reward accounts in addition to
-/// ordinary ones.
+/// Helper type for providing a way to represent virtual reward/accrue accounts
+/// in addition to ordinary ones.
 pub enum Address {
```

Comment:
I'm not sure I understand why there is no virtual foundation accrue account.

The foundation account is a real account, but it is updated once per payday. During the payday amounts are accrued to it but they are not actually owned by it at the time.